fixtures/nested-1/hello

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env sh
-# SUMMARY: Prints "hello"
+#!/usr/bin/env bash
+
+# Don't include in menu_test.go
+if [[ "$1" == "--summary" ]]; then exit 0; fi
```

Comment:
Do you want to leave these as `sh` and do this comparison without the double brackets?

Reply:
Yep! I made that change.